wren-ui/src/components/table/MultiSelectBox.tsx

```diff
@@ -91,8 +97,24 @@ export default function MultiSelectBox(props: Props) {
       <Table
         rowSelection={{
           type: 'checkbox',
-          selectedRowKeys,
-          onChange: onSelect,
+          selectedRowKeys: Array.from(selectedRowKeys),
+          onSelect: (record) => onSelect(record['value']),
+          onChange(keys) {
+            // if more than 1 rows selected and there is only one possibility, if user selects all rows.
```

Comment:
hmm... I thought `onSelectAll` is simpler, it only executes when you click all rows checkbox.
May I ask the reason why we use `onChange` to handle it? 🤔 

<img width="1122" alt="image" src="https://github.com/Canner/WrenAI/assets/9657305/fc41cc93-4fd3-4942-8fa2-89dc20bdf324">

Reply:
onSelectAll is deprecated that's why I used onChange.

Reply:
Noted. Thanks for telling me.